### PR TITLE
feat(Typography): remove small prop from paragraph (p)

### PR DESCRIPTION
**Pull request by joakbjerk**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx [hunk 1]
```diff
@@ -288,4 +290,10 @@ When you convert from `<Modal />` or `<Modal mode="dialog" />` to `<Dialog />` 
 
 1. The FormRow properties `indent` and `indent_offset` were removed.
 
+## Element changes
+
+### [Paragraph](/uilib/elements/paragraph/)
+
+`small` removed as prop since the component already offers two different ways of setting sizes through the use of `size`and `modifier`.
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx:297]: I like your text, but it sounds more like a PR description to me. I would rather not mention arguments in this doc. Look at it more as a "to do" list.

```suggestion
Removed `small` as prop. Use `size="small"` instead.
```

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx [hunk 2]
```diff
@@ -288,4 +290,10 @@ When you convert from `<Modal />` or `<Modal mode="dialog" />` to `<Dialog />` 
...
+
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v10-info.mdx:298]:
```suggestion
1. Find the `small ` property and replace it with `size="small"`.
```

**Review by langz — APPROVED** [review]
Great great 👍

**Review by tujoworker — APPROVED** [review]
Nice work 👍 I have renamed the PR from chore to `feat` so it appears in the release log.

**Pull request merged**